code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1064,14 +1064,9 @@
 			user << "You remove \the [broken_device]."
 			user.put_in_active_hand(broken_device)
 
-	if(ishuman(user))
-		if(istype(user:gloves, /obj/item/clothing/gloves/space_ninja)&&user:gloves:candrain&&!user:gloves:draining)
-			call(/obj/item/clothing/gloves/space_ninja/proc/drain)("CYBORG",src,user:wear_suit)
-			return
-		else
-			if (user:a_intent == I_HELP)
-				help_shake_act(user)
-			return
+	if (user:a_intent == I_HELP)
```

Comment:
>:

a_intent is a MOB VAR. There was no reason for the oldcode to use :, and you should always remove : where you see it.
`if(user.a_intent == I_HELP)`